R/conga.R

```diff
@@ -1,16 +1,18 @@
-#' Calculate continuous overall net glycemic action (CONGA)
+#' Continuous Overall Net Glycemic Action (CONGA)
 #'
 #' @description
 #' The function conga produces a CONGA values a tibble object.
```

Comment:
**in** a nibble object? I will also recommend just saying "for any n hours apart" (rather now supports)